actions/online-sign/action.yml

```diff
@@ -0,0 +1,71 @@
+name: 'Online sign'
+description: 'Creates a snapshot and timestamp if needed, moves publish branch if needed'
+
+inputs:
+  force_publish:
+    description: 'Publish even when snapshot and timestamp are not created'
+    required: true
```

Comment:
As mentioned in the tuf-on-ci-template comment: Maybe this is not needed and the action should figure this out: if `$GITHUB_EVENT_NAME` is not "schedule" we want force_publish.

Reply:
This is done: I will leave this unresolved until I've seen a scheduled workflow work correctly